.eslintrc

```diff
@@ -32,6 +32,7 @@
   "rules": {
     // Plugins
     "import/no-unresolved": "error",
+    "import/extensions": ["error", "always", {ignorePackages: true}],
```

Comment:
Please make sure to sort the entries alphabetically in each section, hence this needs to be moved up.

Furthermore, please make sure to format this object consistently with the rest of the file; see for example https://github.com/mozilla/pdf.js/blob/40f531ee873f523e2991fd6c2ca00a336fdbfbdd/.eslintrc#L46